packages/components/src/Tooltip/Tooltip.tsx

```diff
@@ -65,11 +64,11 @@ export function Tooltip({ message, children }: TooltipProps) {
               exit="startOrStop"
               transition={{
                 type: "spring",
-                damping: 20,
-                stiffness: 300,
+                damping: 50,
+                stiffness: 500,
               }}
             >
-              <Text>{message}</Text>
+              <p className={styles.tooltipMessage}>{message}</p>
```

Comment:
I wonder if this type of text is something that we will be seeing more in the future and if it's worth making a new variation of text called "reverse".

Reply:
That actually might be helpful, I think it might need a bit more thought (is it a variant of text, new component, etc.) than the scope of this PR because this classname also handles _font weight_, so does this become a new weight variant of Text? (and then what do we name that property, etc).

The `reverseTheme` part is a Just Do It in my opinion, but I'm not super equipped to wrangle the props in Text on my own 😅